data_safe_haven/commands/sre.py

```diff
@@ -1,97 +1,21 @@
-"""Command-line application for deploying a Data Safe Haven component, delegating the details to a subcommand"""
-
 from typing import Annotated, Optional
```

Comment:
Can we restore a module-level docstring?

Reply:
Done in aeb6e6fa